### ISSUE TITLE
Export DB option only works when an external storage is present in the device

### ISSUE THREAD

**Issue by viggyprabhu** [Freeyourgadget/Gadgetbridge#622]
When I tried exporting DB on my phone without an external storage, even though I had enough memory in my phone, I got an error saying, 'Cannot find export path'. This was probably because there was no /storage/sdcard0/ directory. However once I inserted an external storage and tried exporting, Android probably created the /storage/sdcard0/ directory and I am able to store the files. Interestingly, /storage/sdcard0/ is still in my local storage, not external storage. I have attached the screenshot of the app, with and without SDCard. 
![without_sdcard](https://cloud.githubusercontent.com/assets/343279/24396572/4c7b922e-13c1-11e7-8735-a0c5505e6535.jpeg)
![with_sdcard](https://cloud.githubusercontent.com/assets/343279/24396573/4c7f649e-13c1-11e7-8cc3-1da992f1d992.jpeg)

It should also be noted that once, the external storage was added and even after removing it, the export was working fine. 

#### Your wearable device is:
MI Band 2
*Please specify model and firmware version if possible*
FW: v1.0.1.34
HW: V0.1.3.2

#### Your android version is:
5.1.1

#### Your Gadgetbridge version is:
0.17.5

**Comment by danielegobbetti** [comment]
Would you mind adding this information to our wiki? Thanks!

**Comment by viggyprabhu** [comment]
Definitely, should I create a new Wiki page itself as to how to export data using the gadgetbridge app and mention this as a note there? Also I thought this was more of a bug because anyone without a sdcard will not be able to export at all.

**Comment by danielegobbetti** [comment]
I'm afraid there's little we can do. See the documentation: https://developer.android.com/reference/android/content/Context.html#getExternalFilesDir(java.lang.String)

A user should be able to mount external storage regardless of the sdcard presence. I use the export functionality on my nexus device without sdcard without any issue. To be honest I do *not* know how to unmount the storage even if I wanted to. :)
